src/Data/PostObjectMutation.php

```diff
@@ -122,19 +122,19 @@ public static function prepare_post_object( $input, $post_type_object, $mutation
 	 * This updates additional data related to a post object, such as postmeta, term relationships,
 	 * etc.
 	 *
-	 * @param int          $post_id               $post_id      The ID of the postObject being
-	 *                                            mutated
-	 * @param array        $input                 The input for the mutation
-	 * @param \WP_Post_Type $post_type_object The Post Type Object for the type of post being
- * mutated
-	 * @param string       $mutation_name         The name of the mutation (ex: create, update,
-	 *                                            delete)
-	 * @param \WPGraphQL\AppContext $context The AppContext passed down to all resolvers
+	 * @param int                                  $post_id               $post_id      The ID of the postObject being
+	 *                                                                    mutated
```

Comment:
this line looks weird too?

Reply:
🤷🏻‍♂️ maying it's just the Github UI that makes it look off?

Reply:
Nah these indeed got messed up. 1 sec will push a fix.